gradoop-data-integration/src/test/java/org/gradoop/dataintegration/importers/rdbmsimporter/RdbmsDataImportTest.java

```diff
@@ -0,0 +1,128 @@
+import org.gradoop.dataintegration.importer.rdbmsimporter.metadata.RdbmsTableBase;
+import org.gradoop.dataintegration.importer.rdbmsimporter.metadata.TableToEdge;
+import org.gradoop.dataintegration.importer.rdbmsimporter.metadata.TableToNode;
+import org.gradoop.flink.io.api.DataSource;
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.List;
+
+public class RdbmsDataImportTest extends GradoopFlinkTestBase {
+
+  private static PostgresWrapper pw;
+  private static Connection con;
+  private static MetaDataParser metadataParser;
+  private static String gdlPath;
+
+  @BeforeClass
+  public static void setUp() throws IOException, SQLException {
+    pw = new PostgresWrapper();
+    pw.start();
+
+    con = DriverManager.getConnection(pw.getConnectionUrl());
+
+    metadataParser = new MetaDataParser(con, RdbmsType.MYSQL_TYPE);
+    metadataParser.parse();
+
+    System.out.println(RdbmsDataImportTest.class
+        .getResource("/data/rdbms/expected/cycleTest.gdl").getFile());
+    
+    gdlPath = RdbmsDataImportTest.class
+        .getResource("/data/rdbms/expected/cycleTest.gdl").getFile();
+  }
+  
+  @Test
+  public void dbMetadataTest() throws SQLException, IOException {
+    List<RdbmsTableBase> tableBase = metadataParser.getTableBase();
+
+    assertEquals("Wrong table count !", 1, tableBase.size());
+  }
+
+  @Test
+  public void tablesToNodesTest() throws Exception {
+    List<TableToNode> tablesToNodes = metadataParser.getTablesToNodes();
+    List<TableToEdge> tablesToEdges = metadataParser.getTablesToEdges();
+
+    assertEquals("Wrong tables to nodes count !", 1, tablesToNodes.size());
+    assertEquals("Wrong tables to edges count !", 1, tablesToEdges.size());
+  }
+
+  @Test
+  public void testConvertRdbmsToGraph() throws Exception {
+
+    // creates embedded, temporary postgresql database
+    String url = pw.getConnectionUrl();
+    String user = "userName";
+    String password = "password";
+    String jdbcDriverPath = RdbmsDataImportTest.class
+        .getResource("/data/rdbms/jdbcDrivers/postgresql-42.2.2.jar").getFile();
+    String jdbcDriverClassName = "org.postgresql.Driver";
+
+    // creates rdbms data import of embedded databse
+    DataSource dataSource = new RdbmsDataSource(url, user, password, jdbcDriverPath,
+        jdbcDriverClassName, getConfig());
+
+    LogicalGraph tempInput = dataSource.getLogicalGraph();
+    FlinkAsciiGraphLoader loader = getLoaderFromFile(gdlPath);
+    LogicalGraph expected = loader.getLogicalGraphByVariable("expected");
+
+    DataSet<Vertex> v = tempInput.getVertices().map(new MapFunction<Vertex, Vertex>() {
```

Comment:
use `LocalCollectionOutputFormat` to receive vertex elements + use further post process to transform the vertices OR refactor your expected graph.

Reply:
What is necessary to execute this test? On my pc it could not found the JDBC driver.

Reply:
Actually there should be a maven dependency for postgresql jdbc driver. But I think i will refactor the test section.